napari_arboretum/plugin.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, viewer: napari.Viewer, parent=None):
         self._viewer.layers.events.changed.connect(self._get_tracks_layers)
 
         # store the tracks layers
-        self._tracks_layers = []
+        self._tracks_layers: list = []
```

Comment:
again here `mypy` needed this typed but I don't know what it has in it

Reply:
This one is the napari layer list: `List[napari.layers.Tracks]`